src/layout/styles/SideBarLayout.scss

```diff
@@ -1,11 +1,16 @@
-@import "../../styles/settings";
+@import '../../styles/settings';
 
 #sidebar-layout {
   padding-left: 0.5em;
```

Comment:
The `em` units are inherited and relative to the parent. This can cause styling issues when a component is nested inside itself or another component. This is a very controversial topic, but I suggest using `rem` instead of `em` as it is only relative to the root element of the page.

Reply:
True